proto/BUILD.bazel

```diff
@@ -39,6 +39,18 @@ go_proto_compiler(
     ],
 )
 
+go_proto_compiler(
+    name = "go_grpc_v2",
```

Comment:
I don't love this name. I want to gently nudge callers towards it without forcing them to deal with the internals of which plugin is which.

Any suggestions would be welcome